cmd/feedback/feedback.go

```diff
@@ -203,22 +242,22 @@ func NewFeedbackCommand(clients *shared.ClientFactory) *cobra.Command {
 // runFeedbackCommand will open the user's browser to the feedback survey webpage.
 func runFeedbackCommand(ctx context.Context, clients *shared.ClientFactory, cmd *cobra.Command) error {
 	if len(SurveyStore) == 0 {
-		clients.IO.PrintInfo(ctx, false, "No surveys currently available; please try again later")
+		clients.IO.PrintInfo(ctx, false, "No feedback options currently available; please try again later")
 		return nil
 	}
 
 	surveyNames, surveyPromptOptions := initSurveyOpts(ctx, clients, SurveyStore)
 
 	if _, ok := SurveyStore[surveyNameFlag]; !ok && surveyNameFlag != "" {
 		return slackerror.New("invalid_survey_name").
```

Comment:
```suggestion
		return slackerror.New(slackerror.ErrInvalidSurveyName).
```

I'm not sure if this is a coded error at the moment, but this can help match formats for quick lookups later:

```diff
- 🚫 Please provide a feedback name or remove the --no-prompt flag
+ 🚫 Please provide a feedback name or remove the --no-prompt flag (invalid_survey_name)
```

Reply:
Great suggestion! I'd like to keep this PR focused on adding the Slack CLI feedback option, but I agree that we should improve the feedback's implementation a bit!

I've pull this suggestion into PR #68, which adds `slackerror.ErrFeedbackNameInvalid` and `slackerror.ErrFeedbackNameRequired`!